SourceX/DiabloUI/diabloui.cpp

```diff
@@ -220,17 +226,21 @@ bool UiFocusNavigation(SDL_Event *event)
 	case SDL_KEYUP:
 	case SDL_MOUSEBUTTONUP:
 	case SDL_MOUSEMOTION:
+#ifndef USE_SDL1
 	case SDL_MOUSEWHEEL:

```

Comment:
maybe implement using an invalid valueike -2 (or -1 if it dosent collide withe a plausable event different in SDL1)

Reply:
I think it'd be a bit confusing for someone expecting these events to work in SDL 1, let's stick with `#ifdefs` for clarity